cishouseholds/pipeline/pipeline_stages.py

```diff
@@ -491,13 +491,20 @@ def lookup_based_editing(
         input file path name for travel_countries corrections lookup file
     edited_table
     """
+
     df = extract_from_table(input_table)
     cohort_lookup = extract_from_table(cohort_lookup_table)
     travel_countries_lookup = extract_from_table(travel_countries_lookup_table)
+
+    for name, table in zip(
```

Comment:
We need to specifically check for duplicates in the join keys that are used for the lookups (not completed record duplicates). See the joins to identify the join keys for each

Reply:
I would implement this as a function that takes a list of keys to check for duplicates, so that it can be reused. We will want to apply this for the rural_urban_lookup in a different stage too, though I can create a different ticket for this